pkg/api/azenvtypes.go

```diff
@@ -91,7 +91,7 @@ var (
 		AzureCNIImageBase:                "mcr.microsoft.com/containernetworking/",
 		MCRKubernetesImageBase:           "mcr.microsoft.com/",
 		EtcdDownloadURLBase:              "mcr.microsoft.com/oss/etcd-io/",
-		KubeBinariesSASURLBase:           "https://acs-mirror.azureedge.net/kubernetes/",
+		KubeBinariesSASURLBase:           "https://kubernetesartifacts.azureedge.net/kubernetes/",
```

Comment:
no, we need to stick toacs-mirror

Reply:
The PR "build: fetch k8s Windows .zip from kubernetesartifacts storage #2655" introduced this change. This is only for Windows to download init zip package for Windows.

Reply:
https://github.com/tamilmani1989/aks-engine/blob/30ecbad52f8802f08a2115cfd1950a5e89a1b93b/pkg/engine/params_k8s.go#L126-L133
```
			if properties.HasWindows() {
				// Kubernetes packages as zip file as created by scripts/build-windows-k8s.sh
				// will be removed in future release as if gets phased out (https://github.com/Azure/aks-engine/issues/3851)
				kubeBinariesSASURL := kubernetesConfig.CustomWindowsPackageURL
				if kubeBinariesSASURL == "" {
					kubeBinariesSASURL = cloudSpecConfig.KubernetesSpecConfig.KubeBinariesSASURLBase + k8sComponents["windowszip"]
				}
				addValue(parametersMap, "kubeBinariesSASURL", kubeBinariesSASURL)
```

Reply:
https://docs.microsoft.com/en-us/azure/aks/limit-egress-traffic

We deliberately keep using acs-mirror to not break our egress promise.

Reply:
Will update it